hoodie-cli/pom.xml

```diff
@@ -167,11 +167,30 @@
       <artifactId>log4j</artifactId>
       <version>${log4j.version}</version>
     </dependency>
+
+    <dependency>
+      <!-- TODO: Need to refactor to move SchemaUtil.readSchemaFromLogFile from hoodie-hive -->
```

Comment:
We need to move readSchemaFromLogFile from hoodie-hive to avoid depending on hoodie-hive unncecessarily

Reply:
yes. but eventually, we probably can add HiveSyncTool cli from the hoodie-cli also.. it will be much easier to use.